### PR TITLE
Move TiFlash troubleshoot part to a new document

### PR DESCRIPTION
**Pull request by leiysky**
### What is changed, added or deleted? (Required)

Move part of a document to another one.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3192
- Other reference link(s):

### CODE REVIEW

File: tiflash/troubleshoot-tiflash.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: Troubleshoot a TiFlash cluseter
+summary: Learn common operations when you troubleshoot a TiFlash cluster.
+category: reference
+aliases: ['/docs/dev/reference/tiflash/troubleshoot/']
```
Comment (TomShawn) [inline tiflash/troubleshoot-tiflash.md:5]: 需要在 TOC 目录中相应位置添加这篇文档的链接。
```suggestion
```

File: tiflash/troubleshoot-tiflash.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+---
+
+## TiFlash troubleshooting
```
Comment (TomShawn) [inline tiflash/troubleshoot-tiflash.md:8]: 文档应从一级标题开始。一级标题应与 metadata 中的标题一致。请相应地减少以下标题的层级。
```suggestion
# Troubleshoot a TiFlash Cluster
```

File: tiflash/troubleshoot-tiflash.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+---
+title: Troubleshoot a TiFlash cluseter
```
Comment (TomShawn) [inline tiflash/troubleshoot-tiflash.md:2]:
```suggestion
title: Troubleshoot a TiFlash Cluster
```

**Comment by leiysky** [comment]
@TomShawn PTAL

**Comment by TomShawn** [comment]
LGTM
@zanmato1984 @kissmydb PTAL

**Pull request merged**